h/services/list_groups.py

```diff
@@ -77,6 +77,24 @@ def session_groups(self, authority, user=None):
 
         return world_group + user_groups
 
+    def associated_groups(self, authority, user=None):
```

Comment:
Since this function always returns an empty list if `user` is `None` and the authority can be determined from the user otherwise via `user.authority`, I think the `authority` parameter is redundant here?